tracer.spec

```diff
@@ -97,7 +85,7 @@ mkdir -p %{buildroot}/%{python3_sitelib}/tracer
 
 cp -a bin/tracer-2 %{buildroot}/%{_bindir}/
 cp -a bin/tracer-3 %{buildroot}/%{_bindir}/
-ln -s %{_bindir}/tracer-2 %{buildroot}/%{_bindir}/tracer
+ln -s %{_bindir}/tracer-3 %{buildroot}/%{_bindir}/tracer
```

Comment:
Hi @FrostyX ! This is a common misunderstanding: If the Python 2 and Python 3 versions of the executable do exactly the same thing, you should only package the Python 3 version. There is no reason to package it twice if it does the same thing.

The link to the Python RPM Porting Guide is for the section where the Python 2 and Python 3 binaries behave differently. For this case, this section is appropriate: http://python-rpm-porting.readthedocs.io/en/latest/application-modules.html